docs/gambit/mutation-verifier.md

```diff
@@ -0,0 +1,108 @@
+# Solidity Mutation Verifier Based on Gambit
+
+This is a mutation verifier which
+ checks that variants of the original
+ solidity program do not pass the specification.
+If a mutated program passes the specification,
+it may indicate that the specification is vacuous or not rigorous enough.
+It uses mutations from the [Gambit](https://github.com/Certora/gambit)
+  mutation generator.
+In the rest of the document,
+  we refer to both the mutation generator and the verifier as Gambit.
+
+## Running the Mutation Verifier
+
+- Example:
+```
+java -ea -jar $CERTORA/certora_jars/MutationTest.jar /path/to/config/file/Example.conf
+```
+
+- Remember to `yarn install` or `npm install` when working
+  on customer code if required by the customer code infra!
+
+- **NOTE: if a path has spaces, put quotes around it to ensure correct parsing by Kotlin's libraries**
+
+- Gambit **does** support `--staging`!
+**However, Gambit currently has trouble with
+`--send_only` and `--cloud` in the run scripts.
+If you have these flags, please remove them for now.**
+Apologies for the temporary inconvenience!
+
+The tool expects a configuration file (extension `.conf` is required)
+which is a JSON object.
+Here is an example configuration file:
+
+```
+{
+    "project_folder" : "Test/10Power",
+    "run_script" : "Test/10Power/runDefault.sh",
+    "gambit" : {
+        "filename" : "Test/10Power/TenPower.sol",
+        "solc" : "solc8.10"
+    },
+    "staging" : "master",
+    "manual_mutations" : {
+        "path/to/original/file.sol" : "path/to/manual/mutations/",
+        ...
+    },
+    ...
+}
+```
+
+Required Keys for the JSON Configuration File:
+- `"project_directory"` : the directory containing the original CVT project on which to perform mutation testing
+- `"run_script"` : the bash script used to run verification on the original project, usually `project_directory/run.sh` or similar
```

Comment:
Does this work if the project is using certora conf files?  E.g. [the ERC20 Example](https://github.com/Certora/ERC20Example/blob/main/certora/scripts/verifyERC20.sh)

Reply:
no, it does not.